android/src/main/new_arch/react/renderer/components/RNLiveMarkdownSpec/MarkdownTextInputDecoratorShadowNode.cpp

```diff
@@ -0,0 +1,125 @@
+#include "MarkdownTextInputDecoratorShadowNode.h"
+
+#include <fbjni/fbjni.h>
+#include <react/fabric/JFabricUIManager.h>
+#include <react/jni/ReadableNativeMap.h>
+#include <react/renderer/components/androidtextinput/AndroidTextInputState.h>
+#include <react/renderer/components/view/conversions.h>
+#include <react/renderer/core/ComponentDescriptor.h>
+#include <yoga/Yoga.h>
+
+namespace facebook {
+namespace react {
+
+extern const char MarkdownTextInputDecoratorViewComponentName[] =
+    "MarkdownTextInputDecoratorView";
+
+void MarkdownTextInputDecoratorShadowNode::initialize() {
+  ShadowNode::traits_.unset(ShadowNodeTraits::ForceFlattenView);
+}
+
+void MarkdownTextInputDecoratorShadowNode::createCustomContextContainer() {
+  static const auto customFabricUIManagerClass = jni::findClassStatic(
+      "com/expensify/livemarkdown/CustomFabricUIManager");
+  static const auto createMethod =
+      customFabricUIManagerClass
+          ->getStaticMethod<JFabricUIManager::javaobject(
+              JFabricUIManager::javaobject, ReadableMap::javaobject, int)>(
+              "create");
+
+  const auto markdownStyle =
+      this->getProps()->rawProps["markdownStyle"];
+  const auto parserId =
+      this->getProps()->rawProps["parserId"].asInt();
```

Comment:
```suggestion
  const auto &rawProps = this->getProps()->rawProps;
  const auto markdownStyle = rawProps["markdownStyle"];
  const auto parserId = rawProps["parserId"].asInt();
```

Reply:
BTW can we use `const auto &markdownStyle` instead of `const auto`?

Reply:
Yes, [3c1be5b](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/3c1be5be4104a1f8259ae41d809fce73bc7ccab8)